src/helpers/visibilityClassnames.ts

```diff
@@ -0,0 +1,7 @@
+import { display } from 'classnames/tailwind'
+
+type Size = 'xs' | 'sm' | 'md' | 'lg'
```

Comment:
You can use `TPseudoClassVariants`, but it doesn't limit you only to sizes, so maybe it's a good idea to use only our own

Reply:
It's better to use only sizes that we need 👍